docs/build.js

```diff
@@ -8,28 +8,33 @@ const DOCS = [
   {
     input: 'src/index.ts',
     output: 'docs/api/papr.md',
-    title: 'Papr'
+    title: 'Papr',
+  },
+  {
+    input: 'src/hooks.ts',
+    output: 'docs/api/hooks.md',
+    title: 'Hooks',
   },
   {
     input: 'src/model.ts',
     output: 'docs/api/model.md',
-    title: 'Model'
+    title: 'Model',
   },
   {
     input: 'src/schema.ts',
     output: 'docs/api/schema.md',
-    title: 'Schema'
+    title: 'Schema',
```

Comment:
😱 !!!!! Is this (and other changes in this PR) coming from a local prettier config? What's going on here?

Reply:
This was auto-formatted by the prettier extension in my editor. It seems we forgot to format the `.js` files in our `yarn pretty` command. I'll update that as well.